integer/tests/log.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::approx_constant)]
```

Comment:
Same here, it's better to just substitute the `core::f64::consts::*` to follow the clippy instructions